xbmc/profiles/ProfileManager.cpp

```diff
@@ -374,6 +375,7 @@ void CProfileManager::FinalizeLoadProfile()
   PVR::CPVRManager &pvrManager = CServiceBroker::GetPVRManager();
   CNetworkBase &networkManager = CServiceBroker::GetNetwork();
   ADDON::CAddonMgr &addonManager = CServiceBroker::GetAddonMgr();
+  ADDON::CBinaryAddonManager &binAddonManager = CServiceBroker::GetBinaryAddonManager();
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types